src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -455,71 +438,40 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
     */
     public static Map<String, String> getLabels(String sObjectName) {
 
-        if (sObjectName == 'Batch_Template__c') {
-            Map<String, String> labels = getTemplateLabels();
-            return labels;
-        } else if (sObjectName == 'DataImportBatch__c') {
+        if (sObjectName == 'DataImportBatch__c') {
             Map<String, String> labels = getBatchLabels();
             return labels;
         } else {
             throw new AuraHandledException('ERROR: ' + 'Only Batch_Template__c and DataImportBatch__c are supported objects.');
```

Comment:
Only DataImportBatch__c is supported now, if you want to keep the fork in logic.